client/src/ts/dd_main.ts

```diff
@@ -248,21 +248,21 @@ ws.onmessage = (event) => {
       break;
     case canDefs.CanMessage.CAN_MESSAGE_AUX_DCDC_VC:
       document.getElementById('aux-current').innerHTML =
-          (msg.data.aux_current / 1000).toPrecision(2) + ' mA';
+          (msg.data.aux_current / 1000).toPrecision(4) + ' mA';
```

Comment:
Seems like `toFixed` might be a better function?

Reply:
Done.